nixos/modules/services/misc/nesthub.nix

```diff
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nesthub;
+in {
+  options = {
+    services.nesthub = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the Nesthub service.";
+      };
```

Comment:
```suggestion
      enable = mkEnableOption "Whether to enable the Nesthub service.";
```

Reply:
https://github.com/NixOS/nixpkgs/blob/780f3cf9facc320dba8709ff4c4caf9f3f94aded/lib/options.nix#L93-L100

We probably don't need the `Whether to enable` prefix and the period if we use `mkEnableOption`.